website/src/components/ContactSupport/index.js

```diff
@@ -3,16 +3,18 @@ import styles from './styles.module.css';
 
 const ContactSupport = () => {
   return (
-    <div className={styles.contact}>
-      <h1 className={styles.contact_title}>Contact support</h1>
-      <p>
-        If you need additional support, one of our knowledgeable support team
-        will be happy to assist.
-      </p>
-      <button href="#" className={styles.contact_button}>
-        raise a ticket
-      </button>
-    </div>
+    <section className={styles.contact_container}>
+      <div className={styles.contact}>
+        <h1 className={styles.contact_title}>Contact support</h1>
+        <p>
+          If you need additional support, one of our knowledgeable support team
+          will be happy to assist.
```

Comment:
just wondering aloud here if maybe we might want a call to action here something along the lines of 
```suggestion
         If you need additional support, one of our knowledgeable support team
          will be happy to assist.
          Please raise a ticket and we will handle this in no time.
```

Reply:
These are just mock texts for now. I assume @norosa will take it and edit with the proper content

Reply:
That's correct. Thanks for your vigilance, @chaim0m! I will take this into account when I turn my attention to copywriting.